bot.py

```diff
@@ -57,7 +57,7 @@ class MusicBot:
     # pylint: disable=no-self-use
     # pylint: disable=too-many-instance-attributes
 
-    COMMAND_PREFIX = "!"
+    COMMAND_PREFIX = "-"
```

Comment:
Unintended change?

Reply:
Even so, I actually like this change. I say we keep it!

Reply:
Yes, this was unintended. I'll leave it if no one objects.